CONTRIBUTING.md

```diff
@@ -356,6 +357,96 @@ splits/joins making it easier to maintain code following the PEP8
 recommendations.  NeuroDebian provides `pycharm-community-sloppy`
 package to ease pycharm installation even further.
 
+### Benchmarking
+
+We use [asv] to benchmark some core DataLad functionality.
+The benchmarks suite is located under [benchmarks/](./benchmarks), and
+periodically we publish results of running benchmarks on a dedicated host
+to http://datalad.github.io/datalad/ .  Those results are collected
```

Comment:
@mih - here is the pointer to our dashboard.  but may be we should add a badge (always green ;-)) to the main README and here pointing to the dashboard